Demo/Demo/SwiftUIComponents/PayPalWeb/PayPalWebStatusView.swift

```diff
@@ -0,0 +1,64 @@
+import SwiftUI
+
+struct PayPalWebStatusView: View {
+
+    var status: Status
+    var payPalViewModel: PayPalWebViewModel
```

Comment:
Why do we need a `Status` in addition to a ViewModel?

Reply:
I think the events are linear and looking at presence of value in event 2 (for instance, order) could mean event 2, 3 or 4.

Reply:
It's used for the switch below